appveyor.yml

```diff
@@ -36,24 +23,25 @@ install:
       https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
       Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
         throw "There are newer queued builds for this pull request, failing early." }
-  - "git clone git://github.com/astropy/ci-helpers.git"
-  - "powershell ci-helpers/appveyor/install-miniconda.ps1"
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "conda create -y -n test python=%PYTHON_VERSION%"
   - "activate test"
-  # Install qtpy from Github
-  - "pip install git+https://github.com/spyder-ide/qtpy.git"
-  # Fix problems with latest pyqt
-  - "conda install pyqt=5.6*"
+  - "conda install -q -y -c spyder-ide --file requirements/conda.txt"
```

Comment:
I think the current version of conda install on AppVeyor is old, do you think that calling `conda update -q conda` is necessary? I'm doing it for my projects... but I don't know if I should or not.

Reply:
If you think this is not, I think I will stop doing that to reduce installation time.

Reply:
It seems newer versions of conda are very, very slow. So I'd prefer to leave it at its current version.

Reply:
Thank you very much for the insight.